src/VSCodeExtension/package-lock.json.v.template

```diff
@@ -6,7 +6,7 @@
     "packages": {
         "": {
             "name": "quantum-devkit-vscode",
-            "version": "0.9999.21061027",
+            "version": "#SEMVER_VERSION#",
```

Comment:
Please make sure that there are no other instances of 0.9999.21061027 left in the file, and the other .template as well. Most likely this got to the file accidentally during a previous commit.

Reply:
Yes, I made sure there is no other instance.